test/generators/whenever_install_generator_test.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require "test_helper"
+require "generators/boring/whenever/install/install_generator"
+
+class WheneverInstallGeneratorTest < Rails::Generators::TestCase
+  tests Boring::Whenever::InstallGenerator
+  setup :build_app
+  teardown :teardown_app
+
+  include GeneratorHelper
+  include ActiveSupport::Testing::Isolation
+
+  def destination_root
+    app_path
+  end
+
+  def test_should_install_whenever_gem
+    Dir.chdir(app_path) do
+      quietly { generator.add_whenever_gem }
+
+      assert_gem "whenever"
+    end
+  end
+
+  def test_should_generate_schedule_file
```

Comment:
Let's update this to "test_should_configure_whenever"

The objective of this test is to run the whole install generator to verify that install generator runs correctly which might not be the case if we run individual methods in each test.

You can refer to the test_should_configure_rswag in the PR https://github.com/abhaynikam/boring_generators/pull/83/files for more details